src/lib/connectors/magicConnectConnector.ts

```diff
@@ -75,23 +75,23 @@ export class MagicConnectConnector extends MagicConnector {
           await magic.auth.loginWithEmailOTP({
             email: output.email,
           });
-        }
```

Comment:
Hey @kesar , thank you for the review ! I think so, yes. 

Otherwise, in the case when a user closes our modal,  `getAddress()` will be called, triggering the opening of the Magic Connect modal which breaks the whole flow in terms of UX.

Instead, the behavior I am proposing is only calling both `loginWithEmailOTP()` & `getAddress()` when the email in the modal has been properly filled. This prevent the opening of the Magic Connect modal when the user closed our custom one.

If my explanation is not clear I'd be glad to try and detail it more.